sadcowpng/sharedspaces/templates/sharedspaces/create_space.html

```diff
@@ -3,8 +3,41 @@
 <head>
     <meta charset="UTF-8">
     <title>Create A Space</title>
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
```

Comment:
Consider breaking up this line into multiple lines for readability, perhaps by html field arguments

Reply:
updated